App.tsx

```diff
@@ -1,19 +1,43 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import OnboardingScreen from './screens/Onboarding/OnboardingScreen';
+//import InformationScreen from './screens/Registration/InformationScreen/InformationScreen'
+//import InterestsScreen from './screens/Registration/InterestsScreen/InterestsScreen'
+
+import * as Font from 'expo-font';
+import React, { useState, useEffect,  } from 'react';
+
+
+
```

Comment:
Since we don't want to display anything until the fonts are loaded, we don't want to return anything. In the meantime, we would like to keep showing a splash screen instead of an empty view. Therefore, we use a function to prevent auto hide of a splash screen that we can control with `expo-splash-screen`. Install the package and call the function.

```suggestion
SplashScreen.preventAutoHideAsync()
```

Reply:
Done!